extensions/shared/paid-blocks/index.js

```diff
@@ -29,6 +30,7 @@ const jetpackPaidBlock = ( settings, name ) => {
 	return {
 		...settings,
 		edit: jetpackPaidBlockEdit( settings.edit ),
+		icon: getPaidIcon( settings, name ),
```

Comment:
I would rather do

`icon: hasPaidIcon ? getPaidIcon( name ) : settings.icon`

Reply:
I originally had it that way too, but I was using `hasPaidIcon` within `getPaidIcon` as well, so it would run twice unnecessarily. I'm fine switching back to this style though if you want!

Reply:
No worries, it is fine